databus-api/src/test/java/com/bazaarvoice/emodb/databus/api/SubscriptionJsonTest.java

```diff
@@ -21,7 +21,7 @@
     @Test
     public void testSubscriptionJson() throws Exception {
         Date now = new Date();
-        String nowString = DateTimeFormatter.ISO_INSTANT.withZone(ZoneOffset.UTC).format(now.toInstant());
+        String nowString = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").withZone(ZoneOffset.UTC).format(now.toInstant());
```

Comment:
how this change is relative to C*/metrics dependencies upgrade?

Reply:
Reverted the change. This test failed couple of times in local hence made this change. Looks like we don't need it.